exchanges/bitstamp/bitstamp.go

```diff
@@ -86,6 +83,40 @@ func (b *Bitstamp) GetFee(ctx context.Context, feeBuilder *exchange.FeeBuilder)
 	return fee, nil
 }
 
+// GetTradingFee returns a trading fee based on a currency
+func (b *Bitstamp) getTradingFee(ctx context.Context, feeBuilder *exchange.FeeBuilder) (float64, error) {
+	fees, err := b.GetAccountTradingFee(ctx, feeBuilder.Pair)
+	if err != nil {
+		return 0, err
+	}
+	if len(fees) != 1 {
+		return 0, errors.New("did not receive exactly one fee from api")
+	}
+	fee := fees[0].Taker
+	if feeBuilder.IsMaker {
+		fee = fees[0].Maker
+	}
+	return fee / 100 * feeBuilder.PurchasePrice * feeBuilder.Amount, nil
+}
+
+// GetAccountTradingFee returns a TradingFee for a pair
+func (b *Bitstamp) GetAccountTradingFee(ctx context.Context, pair currency.Pair) ([]MakerTakerFees, error) {
+	path := bitstampAPITradingFees + "/" + strings.ToLower(pair.String())
+
+	var resp TradingFees
+	err := b.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, path, true, nil, &resp)
+	return resp.Fees, err
```

Comment:
Please return `resp`, not `resp.Fees` to not filter out response API data

Reply:
Completed in b0f9d3